snippets/standard/datetime/json/custom-writing-with-utf8jsonwriter.cs

```diff
@@ -17,10 +17,15 @@ public static void Main(string[] args)
         {
             using (var writer = new Utf8JsonWriter(stream, options))
             {
-                string dateStr = DateTime.UtcNow.ToString("F", CultureInfo.InvariantCulture);
+                DateTime utcNow = DateTime.UtcNow;
+                string dateStr = utcNow.ToString("F", CultureInfo.InvariantCulture);
 
                 writer.WriteStartObject();
-                writer.WriteString("date", dateStr);
+                writer.WritePropertyName("dateFormats");
+                writer.WriteStartArray();
+                writer.WriteStringValue(utcNow);
```

Comment:
Hmm, re-reading the current doc again, looks like we already show-cased the default reader/writer behavior with a sample up-front. I am on the fence of this change on the writer and **would be OK with just leaving things as is** (or showing both formats explicitly). When reviewing previously I hadn't realized those cases were already covered int he doc.

So, we could just close this PR and revert https://github.com/dotnet/docs/pull/14056/commits/5829ca43e2eeccfeb10f5e631852e9bf71e9ae3b.